app/src/main/java/com/github/sdp/mediato/model/User.java

```diff
@@ -0,0 +1,127 @@
+package com.github.sdp.mediato.model;
+
+import com.github.sdp.mediato.errorCheck.Preconditions;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class User {
+    public static final int LIMIT_LOCATION_SIZE = 2;
+    public static final String FAVOURITES_COLLECTION = "Favourites";
+    public static final String RECENTLY_WATCHED_COLLECTION = "Recently Watched";
+
+    private String id = "";
+    private String username = "";
+    private String displayedName = "";
+    private String email = "";
+    private String birthDate = "";
+    private String registerDate = "";
+
+    private List<Double> location = new ArrayList<>();
+    private List<User> friends = new ArrayList<>();
+    private Map<String, List<Review>> reviews = new HashMap<>();
+    private User(){}
+    public User(UserBuilder builder){

```

Comment:
I think this constructor should be private if we are using the Builder design pattern.